docs/_static/style.css

```diff
@@ -3,3 +3,15 @@
 .wy-side-nav-search {
   background-color: #fff;
 }
+
+.beginner-guide {
+  background-color: #4e96fd;
+}
+
+.advanced-guide {
+  background-color: #00a99a;
+}
+
+.contributor-guide {
+  background-color: #ab00b6;
+}
```

Comment:
I'd probably lighten these background colors to make it easier to read the text (perhaps use RGBA with an opacity of 50%?)

Reply:
Great idea! Added